packages/combobox/src/Combobox.ts

```diff
@@ -306,17 +339,13 @@ export class Combobox extends Textfield {
                 @sp-closed=${this.handleClosed}
                 @sp-opened=${this.handleOpened}
                 type=${this.type}
-                aria-describedby=${this.helpTextId}
-                aria-label=${ifDefined(this.label || this.placeholder)}
-                aria-invalid=${ifDefined(this.invalid || undefined)}
                 maxlength=${ifDefined(
                     this.maxlength > -1 ? this.maxlength : undefined
                 )}
                 minlength=${ifDefined(
                     this.minlength > -1 ? this.minlength : undefined
                 )}
                 pattern=${ifDefined(this.pattern)}
-                placeholder=${ifDefined(this.placeholder)}
```

Comment:
Did we intend to remove this here?

Reply:
Oh, yeah, I did. There's no placeholder text for the combobox. We didn't even have `this.placeholder` defined, iirc

Reply:
I thought [this](https://github.com/adobe/spectrum-web-components/blob/main/packages/textfield/src/Textfield.ts#L77) would apply here, but maybe not. As much a design and accessibility speak against placeholder content, many consumers want control over it, so we likely should include it when possible.

Reply:
The point I'm trying to make is that a combobox doesn't have placeholder content to begin with. It's not part of the design spec or the WAI ARIA spec...

Reply:
Ah yes, `placeholder` is a part of the underlying `<input type="text">` https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder, for good or ill people like to use it...